cms/jinja2/templates/base.html

```diff
@@ -174,7 +175,7 @@
     onsCookiesBanner({
       'lang': "cy" if LANGUAGE_CODE == "cy" else "en",
       'serviceName': COOKIE_BANNER_SERVICE_NAME,
-      'settingsLinkUrl': MANAGE_COOKIE_SETTINGS_URL,
+      'settingsLinkUrl': "/cookies",
```

Comment:
Question: How do we manage this for the Welsh locale? Once we have subdomains, this will work, but until then I guess this will only work for English?

Reply:
Addressed in [46550b4](https://github.com/ONSdigital/dis-wagtail/pull/375/commits/46550b468ae6cf9c018055544c8f3ef0cb56378d)